LEGO1/lego/legoomni/include/legometerpresenter.h

```diff
@@ -4,7 +4,53 @@
 #include "mxstillpresenter.h"
 #include "mxstring.h"
 
+struct Rect16 {
+	// FUNCTION: BETA10 0x10097ee0
+	Rect16(){};
+
+	// FUNCTION: BETA10 0x100981f0
+	inline void SetLeft(MxS16 p_value) { m_left = p_value; }
```

Comment:
Minor nitpick, could you name the parameters in sync with the members they set? like `p_left`, `p_top` etc.